### PR TITLE
Clayton

### PR DESCRIPTION
**Pull request by marsha88**
Updated TDPAlg.js output schema, added Tests folder, and wrote all functions to get specific graph information from the output of TDPAlg.calc().

### CODE REVIEW

File: tests/TDPAlgTests/generateGraphData.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+	for(i = 0; i < outputArray.length; i++){
+		if(typeof outputArray[i] !== "undefined"){
+			outputArray[i][0] = increments[i];
+			outputArray[i][1] /= allYears.length;
+			outputArray[i][2] /= allYears.length;
+		}
+	}
+
+	return outputArray;
+}
```
Comment (jonahheeren) [inline tests/TDPAlgTests/generateGraphData.js]: Semicolon

File: tests/TDPAlgTests/generateGraphData.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+		//set each month to numerical value starting from 0		
+		if(typeof outputArray[i] !== "undefined"){
+			outputArray[i][0] = i;	
+			outputArray[i][1] /= allYears.length;
+			outputArray[i][2] /= allYears.length;
+		}
+	}
+
+	return outputArray;
+}
```
Comment (jonahheeren) [inline tests/TDPAlgTests/generateGraphData.js]: Semicolon

File: tests/TDPAlgTests/generateGraphData.js [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+	for(var i = 0; i < outputArray.length; i++){
+		//set each month to numerical value starting from 0
+		if(typeof outputArray[i] !== "undefined"){
+			outputArray[i][0] = i;
+		}
+	}
+
+	return outputArray;
+}
```
Comment (jonahheeren) [inline tests/TDPAlgTests/generateGraphData.js]: Semicolon

File: tests/TDPAlgTests/generateGraphData.js (hunk 1 above)
Comment (marsha88) [inline tests/TDPAlgTests/generateGraphData.js]: fixed

**Pull request merged**